.github/workflows/temp-integration-test.yml

```diff
@@ -0,0 +1,25 @@
+name: TEMP OpenCue Integration Test
```

Comment:
This is a temporary workflow for demonstrating the integration test as part of this PR. It will be removed prior to merging -- IMO we don't need to be running this for every PR, only on merge to master.